lib/saluki-context/src/tags/raw.rs

```diff
@@ -130,3 +131,64 @@ fn split_at_delimiter(input: &str, delimiter: u8) -> Option<(&str, &str)> {
         }
     }
 }
+
+/// A wrapper over a `RawTags` iterator that allows for additional tags to be appended to the end of the iterator.
+#[derive(Clone)]
+pub struct TagsWrapper<'a> {
```

Comment:
Can we call this `ChainedRawTags`?

The doc comment is fine, but I'd just like to see the type name to slightly more descriptive.